external/libcoap/Makefile

```diff
@@ -72,8 +72,15 @@ CSRCS += str.c
 CSRCS += subscribe.c
 CSRCS += uri.c
 
+# libcoap flags
+CFLAGS += -DWITH_POSIX
+CFLAGS += -D__TINYARA__
+ifeq ($(CONFIG_NETUTILS_LIBCOAP_DEBUG),n)
+CFLAGS += -DNDEBUG
+endif
```

Comment:
These, WITH_POSIX, __TINYARA__ and NDEBUG, are dupplicated with [[config.h]](https://github.com/Samsung/TizenRT/blob/master/external/include/protocols/libcoap/config.h).
Almost definitions will be replaced (re-defined).

Reply:
ok I will remove